libs/model/src/contest/GetThreadContestManagers.query.ts

```diff
@@ -0,0 +1,38 @@
+import { Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { QueryTypes } from 'sequelize';
+import { sequelize } from '../database';
+
+export function GetThreadContestManagers(): Query<
+  typeof schemas.GetThreadContestManagers
+> {
+  return {
+    ...schemas.GetThreadContestManagers,
+    auth: [],
+    secure: false,
+    body: async ({ payload }) => {
+      const contestManagers = await sequelize.query<{
+        contest_address: string;
+      }>(
+        `
+            SELECT
+              cm.contest_address
+            FROM "Communities" c
+            JOIN "ContestManagers" cm ON cm.community_id = c.id
+            JOIN "ContestTopics" ct ON cm.contest_address = ct.contest_address
+            WHERE ct.topic_id = :topic_id
+            AND cm.community_id = :community_id
+            AND cm.cancelled = false
```

Comment:
are these filters excluding ended contests?

Reply:
Added condition for it.

Note– `ended` only applies for one-off contests since we flag it as `true` during rollover. For recurring, it's never in an "ended" state unless explicitly cancelled.